### ISSUE TITLE
QCed data not showing up

### ISSUE THREAD

**Issue by hydr0boy** [NMBGMR/wellpy#29]
The data that has already been QCed is not showing up in the initial viewer when appending new data.
It also does not show up in the 'Viewer tab'. Only the original 'head' or the blue line shows up.

**Comment by jirhiker** [comment]
without an example i can't test

**Comment by hydr0boy** [comment]
Yup all fixed, thanks! This seems silly but could we change the colors around. I use your 'save PDF' button a lot to give Stacy quick print outs when she asks for them, but now the QCed data is green. Could I get the QC'ed data to show up Black
UnQC'ed data Red
Brand new data come in Green
The head data can stay blue
